plugins/woocommerce-admin/client/products/sections/product-details-section.tsx

```diff
@@ -141,6 +149,14 @@ export const ProductDetailsSection: React.FC = () => {
 							}
 						/>
 					) }
+					<RichTextEditor
+						label={ __( 'Description', 'woocommerce' ) }
+						blocks={ descriptionBlocks }
+						onChange={ ( blocks ) => {
+							setDescriptionBlocks( blocks );
+							setValue( 'description', serialize( blocks ) );
```

Comment:
How to deal with validations here? If we are going to use the form validations then this component must be tied to the form system. So `onBlur` is required also the `className` and `help` returned by the `getInputProps`.

Reply:
Nice!  I like the additions of `onBlur` and `className`.  While we don't have any validation needed yet for this component, I think those are worthwhile in general to have.

As for `help`, I think we might hold off on this for now since we don't yet have a need for it (but probably will soon).  I think one of the issues with help text is figuring out placement. Since the editor can expand vertically beyond the viewport, help text might never be seen.

The help text is probably an issue we can tackle in a follow up with some design input.  cc @jarekmorawski

Reply:
After tinkering with the `onBlur` for a bit, it seems there is no `onBlur` available in the block editor provider.  This is most likely because specific content editable areas or blurred consistently when moving from block to block.

I think that we can probably implement some kind of click outside detection and try to catch when deselection occurs (would need to be debounced to avoid false positives between block selection), but I'd recommend we push these to another PR since they aren't needed for the functionality in this one.